rector.php

```diff
@@ -0,0 +1,31 @@
+<?php
+declare(strict_types=1);
+
+use Rector\Core\Configuration\Option;
+use Rector\Php74\Rector\Property\TypedPropertyRector;
+use Rector\Set\ValueObject\LevelSetList;
+use Symfony\Component\DependencyInjection\Loader\Configurator\ContainerConfigurator;
+
+return static function (ContainerConfigurator $containerConfigurator): void {
+    // get parameters
+    $parameters = $containerConfigurator->parameters();
+    $parameters->set(Option::PATHS, [
+        __DIR__ . '/src'
+    ]);
+    $parameters->set(Option::SKIP, [
+        \Rector\Php71\Rector\ClassConst\PublicConstantVisibilityRector::class,
+    ]);
+
+    // Define what rule sets will be applied
+    $containerConfigurator->import(LevelSetList::UP_TO_PHP_72);
+    $containerConfigurator->import(LevelSetList::UP_TO_PHP_81);
+
+    // is your PHP version different from the one your refactor to? [default: your PHP version], uses PHP_VERSION_ID format
+    $parameters->set(Option::PHP_VERSION_FEATURES, \Rector\Core\ValueObject\PhpVersion::PHP_72);
+
+    // get services (needed for register a single rule)
+    // $services = $containerConfigurator->services();
+
+    // register a single rule
+    // $services->set(TypedPropertyRector::class);
```

Comment:
Do we need to keep the commented lines of code?

Reply:
Nope.  I can remove them.  These just came from `vendor/bin/rector init`, a way to automatically create a Rector configuration file (which this is).